src/com/hms/flexyosisoftconnector/Tag.java

```diff
@@ -120,6 +170,264 @@ public void recordTagValue(String time)
       }
    }
 
+   /**
+    * exportBlockDescriptor
+    * send an EBD command with a start and end time and a tag name.
+    * start time is stored in an external file, end time is current time.
+    */
+   public void exportBlockDescriptor()
+   {
+      // time pointers to keep track of start and end of queue
+      String startTime = "null";
+      String endTime = "null";
+
+      // get the time pointer to use in EBD call for the start of historical logs
+      startTime = readTimePointerFile();
+
+      // check that the error value did not get returned. Log if it did, and do not make EBD call.
+      if (startTime.equalsIgnoreCase("-1")) {

```

Comment:
-1 should be a defined as a const if it means an error

const int TIME_ERROR = -1;